python/pysrc/kaskada/_timestream.py

```diff
@@ -760,6 +774,12 @@ def max(self, window: Optional[kd.windows.Window] = None) -> Timestream:
 
         Args:
             window: The window to use for the aggregation. Defaults to the entire Timestream.
+
+        See Also:
+            This returns the the maximum of values in a
+            column up to and including the current row. See the
+            comparison :func:`greatest` to get the greatest value
```

Comment:
Maybe just `See :func:`greatest` for the greatest value between ...` (specifically, not calling it a comparison). Maybe also put `Timestreams` in source ticks.

Reply:
Maybe also call it "to get the maximum value ..." so it isn't "greatest returns the greatest"?